src/app/_components/ui/storage-location-selector/index.tsx

```
@@ -0,0 +1,217 @@
+  const t = useTranslations();
+
+  const [open, setOpen] = useState(false);
+  const [newLocationName, setNewLocationName] = useState("");
+  const [isCreating, setIsCreating] = useState(false);
+
+  const selectedLocation = locations.find(
+    (loc) => loc.id === selectedLocationId,
+  );
+
+  const handleCreateNew = async () => {
+    if (!newLocationName.trim() || !onCreateNew) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)